### PR TITLE
Add intake & return status times & EIP status return times to anonymized CSV download

### PR DESCRIPTION
**Pull request by alaroia-cfa**
Resolves https://www.pivotaltracker.com/story/show/173933593

Partially-co-authored-by: Ben Golder <bgolder@codeforamerica.org>

### CODE REVIEW

File: app/services/anonymized_intake_csv_service.rb [hunk 1]
```diff
@@ -80,8 +80,39 @@ def decorated_intake(intake)
...
+        intake_status_timestamps[ticket_status.intake_status] = ticket_status.created_at
+      end
+
+      unless return_status_timestamps.key?(ticket_status.return_status)
+        return_status_timestamps[ticket_status.return_status] = ticket_status.created_at
+      end
+
+      unless eip_status_timestamps.key?(ticket_status.eip_status)
+        eip_status_timestamps[ticket_status.eip_status] = ticket_status.created_at
+      end
```
Comment (jenny-heath) [inline app/services/anonymized_intake_csv_service.rb:108]: i may be confused about something here - could there be multiple ticket statuses of a given type? if so, are you just recording the first one encountered? why?

File: spec/services/anonymized_intake_csv_service_spec.rb [hunk 2]
```diff
@@ -45,6 +45,127 @@
...
+          intake_status: EitcZendeskInstance::INTAKE_STATUS_IN_REVIEW,
+        )
+      end
+
+      it "stores timestamps in the appropriate column" do
+        row_1_csv_data = csv[0]
+        expect(row_1_csv_data["Intake Status - Gathering Documents"]).to eq("2020-01-01 00:00:00 UTC")
+        expect(row_1_csv_data["Intake Status - In Review"]).to eq("2020-01-02 00:00:00 UTC")
+        row_2_csv_data = csv[1]
+        expect(row_2_csv_data["Intake Status - Gathering Documents"]).to be_nil
```
Comment (jenny-heath) [inline spec/services/anonymized_intake_csv_service_spec.rb:83]: it seems that in my previous comment i was wrong but i can't quite tell why...

File: app/services/anonymized_intake_csv_service.rb (hunk 1 above)
Comment (alaroia-cfa) [reply app/services/anonymized_intake_csv_service.rb:108]: Good question. I'll push a follow-up commit to this PR that hopefully answers this. Happy to discuss.

File: spec/services/anonymized_intake_csv_service_spec.rb (hunk 2 above)
Comment (alaroia-cfa) [reply spec/services/anonymized_intake_csv_service_spec.rb:83]: The answer is in `verified_change: false` on line 67

https://github.com/codeforamerica/vita-min/pull/441/files#diff-9b3aa8c374ae5eda5bb4a7e87050bb17R67

Ben told me this is important, but I can't quite figure out why, so I'm going to retain the behavior and document in a follow-up commit.

**Pull request merged**